plugins/woocommerce/tests/e2e-pw/tests/customize-store/assembler/footer.spec.js

```diff
@@ -146,8 +146,8 @@ test.describe( 'Assembler -> Footers', { tag: '@gutenberg' }, () => {
 			.all();
 
 		for ( const footerPicker of footerPickers ) {
-			await footerPicker.waitFor();
-			await footerPicker.click();
+			await footerPicker.waitFor( { state: 'visible' } );
+			await footerPicker.click( { delay: 300 } );
```

Comment:
Why is the test failing? I'm trying to understand why adding a delay would help.

Reply:
I think it's a race condition. The test is very flaky these days, and I used the fastest solution. The preview page behaves differently due to very fast PW. There were no responses to wait for.

Reply:
I'm going to rerun core e2e tests 1/6 shard a few times to see if it solves flakiness

Reply:
Worked great in CI after rerunning 5-6 times